src/PostgREST/Config.hs

```diff
@@ -73,6 +74,7 @@ data AppConfig = AppConfig {
   , configJwtAudience       :: Maybe StringOrURI
 
   , configPool              :: Int
+  , configPoolTimeout       :: Float
```

Comment:
Hmm. I don't think the users would need a timeout like `11.7` seconds.
And when I try a float(`30.5`), the timeout always gets the default `10` seconds.

Converting this into `Int` would be better.

Reply:
Hm, kind of curious to know where things aren't going as expected.

But, I won't push further. As I've mentioned above, I had added float support merely for that piece in hasql docs. 
I've added a commit to use `Int` instead.